content/yaml-notification/publish-release-notes.md

```diff
@@ -96,3 +96,39 @@ There are three supported options to set up release notes:
 ]
 {{< /highlight >}}
 
+### Generating release notes with git commits
+If your Codemagic workflow is triggered when creating a `Git tag`, you may want to automate the process of generating release notes based on your Git commits.
+You can use the **git log** command to generate release notes with commit massages between two Git tags. Here's how you can do it:
+
+{{< highlight yaml "style=paraiso-dark">}}
+  scripts:
+    name: Generating release notes with git commits
+    script: | 
+        git fetch --all --tags
+
+        prev_tag=$(git for-each-ref --sort=-creatordate  --format '%(objectname)' refs/tags | sed -n 2p )
+        notes=$(git log --pretty=format:"\n- %s" "$prev_tag"..HEAD)
+
+        echo "$notes" | tee release_notes.txt
+{{< /highlight >}}
+
+If you use this script localy it will generate release notes with all commits between tags e.x v1.0.0 and v2.0.0
```

Comment:
If you use this script localy, it will generate release notes with all the commits between tags e.g v1.0.0 and v2.0.0